### DISCUSSION TITLE
products auto slide features add.

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5731] [New Feature/Ideas Requests] [answered]
This is a very nice feature. Please add. 

product layout auto sliding feature add, products automatic slider. see the video... same this feature add

part 1: https://streamable.com/sl13e0
part 2: https://streamable.com/9fmd3x

**Comment by chungxon** [comment, marked as answer]
This feature was added. Thank you for your idea!
